ValveStrCpy.h

```diff
@@ -0,0 +1,57 @@
+/** @file
+    @brief Header
+
+    @date 2016
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+//               Copyright Sensics, Inc. 2016.
+//  Distributed under the Boost Software License, Version 1.0.
+//     (See accompanying file LICENSE_1_0.txt or copy at
+//           http://www.boost.org/LICENSE_1_0.txt)
+#ifndef INCLUDED_ValveStrCpy_h_GUID_E8F022B2_8826_40B2_E94E_AA1F2A33ED74
```

Comment:
@toxicFork You can use the OSVR boilerplate generator: <http://tools.getosvr.org/boilerplate/>.